packages/merkle-box-lib/src/addresses.json

```diff
@@ -1,4 +1,3 @@
 {
-  "mainnet": "0xe67516417a934b27cf0c14868f8165b1bc94bf73",
-  "ropsten": "0x0EFbFE4979da0Cb457a6a80Db6F3da6861E120fc"
+  "mainnet": "0xc2dA346E2f655B1013cB0f405Ca1b99553F7a580"
```

Comment:
I mean... strictly speaking, the current Hemi iteration is a Testnet one, not a Mainnet one. 
Where is this `mainnet`  picked up from?

Reply:
Made this configurable via envs here: https://github.com/hemilabs/pure.finance/pull/2/commits/1c0a792be1d0b294cee93ac808edb62ac1b226a9